dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -400,38 +461,28 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglishAndSpanish_ViewEngl
                         .setPageMode(PageMode.LIVE)
                         .build(),
                 mockRequest, mockResponse);
-        Assert.assertTrue("ESP = "+html , html.contains("content3content2Spa"));
+        Assert.assertTrue("ESP = "+html , html.contains("content3Spacontent2Spa"));
 
     }
 
     /**
-     * ContentFallback False
-     * PageFallback True
-     *
-     * Page Spanish
-     *
-     * English -> 404
-     * Spanish -> 2 & 3
-     *
-     * @throws Exception
+     * Method to test: {@link HTMLPageAssetRenderedAPI#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
+     * When: DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE is set to false and DEFAULT_PAGE_TO_DEFAULT_LANGUAGE is set to true
+     *       And the page have version in ESP
+     *       And the page have tree content, where: content1 is just in ENG version, content2 is in ENG and ESP version, content 3 is just in ESP version
+     * Should: If the page is requests in ENG version it should be thrown a {@link HTMLPageAssetNotFoundException}
+     *         If the page is requests in ESP version it should be render with content3 and content2 (both in ESP version)
      */
-    @Test (expected = HTMLPageAssetNotFoundException.class)
-    public void ContentFallbackFalse_PageFallbackTrue_PageSpanish_ViewEnglish404_ViewSpanishContent2And3() throws Exception{
+    @UseDataProvider("cases")
+    public void ContentFallbackFalse_PageFallbackTrue_PageSpanish_ViewEnglish404_ViewSpanishContent2And3(final Container container, final Template template) throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)